### PR TITLE
Update/package versions 10 2022 – combined dependabot upgrades

### PR DESCRIPTION
**Pull request by sampsapenna**
### Description

Aggregated dependabot upgrades, contains all that were opened before 2022-10-28.

### Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update (this needs a follow up PR)

### Changes Made

* Update node packages, python packages, docker images
* Deprecate `.value` from database queries, due to library changes

### Testing

- [ ] Unit Tests
- [ ] Integration Tests
- [x] Tests do not apply
- [ ] Needs testing (start an issue or do a follow up PR about it)

### CODE REVIEW

**Comment by sampsapenna** [comment]
> @sampsapenna I think the errors for Spelling Errors and Vue Unit Test could be fixed in this case.

@hannyle Yep, I'll check those out and see what's breaking 👍🏼

**Comment by blankdots** [comment]
btw there is also #750, which might help streamline the dependency update

**Comment by sampsapenna** [comment]
> btw there is also #750, which might help streamline the dependency update

@blankdots I'm aware, it's getting merged after this one 👍🏼 There were some breaking changes in a couple of npm updates, so I decided to merge them this way first. But it seems a real improvement for the future

File: dockerfiles/Dockerfile-build-crypt [hunk 1]
```diff
@@ -87,7 +87,7 @@ RUN export EMCC_CFLAGS="-I/emsdk/upstream/include -L/emsdk/upstream/lib" \
     && emmake make
 
 # Build UI with encryption
-FROM node:16.15.1-alpine3.15 as FRONTEND
+FROM node:19.0.0-alpine3.15 as FRONTEND
```
Comment (blankdots) [inline dockerfiles/Dockerfile-build-crypt:90]: are you sure you want to go node:19:00 ?

Comment (sampsapenna) [reply dockerfiles/Dockerfile-build-crypt:90]: Good catch, managed to miss that somehow 😅 We most definitely don't, but I'll see if 18 (current LTS) builds

**Comment by csc-felipe** [comment]
Rebasing should solve the python tests, fixed on #824

**Pull request closed without merging**